app/webpacker/controllers/primary_details_controller.js

```diff
@@ -0,0 +1,25 @@
+import { Controller } from "stimulus"
```

Comment:
This file is not formatted as it should be. I'm wondering then how you was able to commit ...

Could you please run `yarn prettier --check .` and copy/paste the output?

Thanks! 🙏

Reply:
I formatted it correctly and pushed jn. Also, I squashed those commits!